frontend/javascripts/oxalis/model/sagas/saga_selectors.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import type { OxalisState } from "oxalis/store";
+import { type Saga, select } from "oxalis/model/sagas/effect-generators";
+import { V2 } from "libs/mjs";
+import { type Vector2, type OrthoView } from "oxalis/constants";
+import { getBaseVoxelFactors } from "oxalis/model/scaleinfo";
+import { getPlaneExtentInVoxelFromStore } from "oxalis/model/accessors/flycam_accessor";
+import Dimensions from "oxalis/model/dimensions";
+
+export function* getHalfViewportExtents(activeViewport: OrthoView): Saga<Vector2> {
```

Comment:
These two functions were part of the old `automatic_brush_saga.js`. I moved them to this file as there are still used.

Reply:
Have you tried using half of the image?